WaypointPairLabeled waypoints
---
std_msgs/Bool arrived